network-api/networkapi/wagtailpages/templates/wagtailpages/campaign_page.html

```diff
@@ -1,6 +1,8 @@
 {% extends "./mini_site_name_space.html" %}
 {% load wagtailcore_tags mini_site_tags %}
 
+{% block h1 %}{% endblock %}
```

Comment:
If the block needs to be set empty, does that mean this was only an issue for campaign pages, or also opportunity pages? (it feels like if we're overriding it here to be empty, and then hardcode it in the subcontent, that block may be in the wrong spot in the parent template).

Reply:
Technically it was only for campaign pages, because it was related to the petition snippet.

That said, I'm pretty sure if this changed happened on opportunity pages, it wouldn't show a visible difference.

If the h1 block is only being used for those two page types, I think I can clean this up.

Reply:
So this isn't an override on the minisitenamespace, it's an override on the modular page, which right now is only used on minisitenamespace, but these comments here concern me: https://github.com/mozilla/foundation.mozilla.org/blob/master/network-api/networkapi/wagtailpages/models.py#L313-L318

I don't want to drift too far from the initial plan here. If modular page was meant to be for primary page too, but there was an issue, I don't want to be making campaign/opportunity based only changes in it.

Reply:
Hitting an issue where I need to know how to reuse small bits of html in django templates. Blocks won't work, and making a whole new file and including it seems super silly. (but might be the expected way)

Reply:
If you're still curious, the fix basically pastes the code in the two places we need it, and removes the block... Like, nothing fancy here.